app/assets/stylesheets/normalize/_import-now.scss

```diff
@@ -0,0 +1,11 @@
+// Import Now
+//
+// If you import this module directly, it will immediately output all the CSS
+// needed to normalize default HTML elements across all browsers.
+//
+// ```
+// @import "normalize/import-now";
+// ```
+
+@import '../normalize';
+@include normalize();
```

Comment:
I would pretty much prefer not dumping third-party assets directly into the app code, and use gems from https://rails-assets.org instead.
It even has [SCSS variant of Normalize](https://rails-assets.org/#/components/normalize-scss)